sfepy/discrete/problem.py

```diff
@@ -645,13 +645,21 @@ def set_bcs(self, ebcs=None, epbcs=None, lcbcs=None):
             conf_ebc = get_default(ebcs, self.conf.ebcs)
             self.ebcs = Conditions.from_conf(conf_ebc, self.domain.regions)
 
+            conf_dgebc = self.conf.get("dgebcs", {})
+            self.ebcs.extend(Conditions.from_conf(conf_dgebc,
+                                                  self.domain.regions))
+
```

Comment:
Is it possible to mix ebcs and dgebcs?

Reply:
No it is not, DGField does not allow setting facet DOFs, because technically there are none. It is however technically possible to mix epbc and dgepbc, although I'm not use if it makes sense.